### DISCUSSION TITLE
[SOLVED] Changing Lazygit doesn't update Neo-tree

### DISCUSSION THREAD

**Discussion by ghost** [nvim-neo-tree/neo-tree.nvim#1490] [Q&A] [answered]
I wanted to post this here as I just found a solution and I'm sure others are experiencing this problem.
This specifically relates to using `lazygit.nvim` and how neo-tree doesn't update after you make git changes through it.

**Comment by ghost** [comment, marked as answer]
Use this below autocommand.
```lua
vim.api.nvim_create_autocmd({ "BufLeave" }, {
	pattern = { "*lazygit*" },
	group = vim.api.nvim_create_augroup("git_refresh_neotree", {clear = true}),
	callback = function()
		require("neo-tree.sources.filesystem.commands").refresh(
			require("neo-tree.sources.manager").get_state("filesystem")
		)
	end,
})
```

**Reply by linux-root** [reply]
I works. My situation is  default config in [LazyVim](https://www.lazyvim.org/extras/editor/neo-tree#neo-treenvim) doesn't work
